doc-content/enterprise-only/ai/ai-credit-card-con.adoc

```diff
@@ -59,7 +59,7 @@ image:ai/dmn.png[]
 
 The input variables are *Age* (cardholder age), *Incident Count* (the number of previous disputes for this cardholder), *Cardholder Status*, and *Fraud Amount*.
 
-{PRODUCT} uses input variables in decision tables to support digital decisioning. The decision tables are created by human business analysts. The data is provided in spreadsheet files that are uploaded to the {CENTRAL} project. The Fortress Bank Credit Card Dispute process has two decision tables, the *Cardholder Risk Rating* table and the *Dispute Risk Rating* table. The *Cardholder Risk Rating* table contains three input variables:  *Incident Count*, *Cardholder Status*, and *Age*. The *Dispute Risk Rating* table contains the *Cardholder Status* input variable. It calculates the risk of the dispute based on the cardholder status and the amount of the dispute.
+{PRODUCT} uses input variables in decision tables to support digital decisioning. The decision tables are created by human business analysts. The analyst creates a draft business requirement analysis document or spreadsheet that stakeholders review and approve. The project designer then uses the {CENTRAL} DMN editor to transfer the data in the analysis document to the DMN model. The Fortress Bank Credit Card Dispute process has two decision tables, the *Cardholder Risk Rating* table and the *Dispute Risk Rating* table. The *Cardholder Risk Rating* table contains three input variables:  *Incident Count*, *Cardholder Status*, and *Age*. The *Dispute Risk Rating* table contains the *Cardholder Status* input variable. It calculates the risk of the dispute based on the cardholder status and the amount of the dispute.
```

Comment:
I believe that "Credit Card Dispute process" should be tagged as it is a UI object name.

Reply:
Kind of my same argument but I'll go with your suggestion.